### ISSUE TITLE
Support tsx

### ISSUE THREAD

**Issue by unlight** [JamesHenry/eslint-plugin-tslint#34]
tslint wrapper consider all files as `ts`
Because of this string:
```
var fakeFilename = 'eslint.ts';

```
which leads to false warning
`no-angle-bracket-type-assertion` (and other) for
```
<main></main>
```
It's not assertion, it's jsx syntax.

**Comment by unlight** [comment]
@JamesHenry #33 will fix the issue
